modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/query/TermsQueryTranslatorImpl.java

```diff
@@ -7,8 +7,11 @@
 
 import com.liferay.portal.search.query.TermsQuery;
 
+import java.util.ArrayList;
+
+import org.elasticsearch.index.query.BoolQueryBuilder;
 import org.elasticsearch.index.query.QueryBuilder;
-import org.elasticsearch.index.query.QueryBuilders;
```

Comment:
@gustavolimav just curious if there was a reason to stop using QueryBuilders? either way is fine, but if QueryBuilders didnt work for some reason i would be interested to know, i probably would have just continued using QueryBuilders

Reply:
No specific reason, I just changed it because it looks easier to understand, but the code is really the same, call `QueryBuilder.termsQuery(name, value)` will just call ` new TermQueryBuilder(name, value)`.

Reply:
I will put QueryBuilders back on the code so we can keep using it